### PR TITLE
changed readme format to markdown

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

**Comment by vinyll** [comment]
Switched to markdown and fixed some typos.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+A zone is a spatial polygon for a given level. It has at least one unique code (unique on its level) and a name. It can have many known keys, that are not necessarily unique (ie. postal codes can be shared by many towns).
+
+Labels are optionally translatable.
+
+Some zones are defined as an aggregation of other zones. They are called _aggregation_ in geozones and builded after all data are loaded.
+
+The following properties are exported in the GeoJSON output:
+
+id
```
Comment (davidbgk) [inline README.md]: I think it will not render as expected, maybe transform to a table?

**Comment by davidbgk** [comment]
Why did you remove more that half of the file?

**Comment by vinyll** [comment]
Ouch! Copy/paste issue. Fixing and updating ;)

**Comment by davidbgk** [comment]
@vinyll I completely forgot that PR, can you resolve conflicts and I'll merge it?

**Comment by vinyll** [comment]
moved to #25

**Pull request closed without merging**